install.sh

```diff
@@ -22,7 +22,7 @@ cat "$GITHUB_WORKSPACE/.github/scripts/conda_build_config.yaml" >> "$PACKAGE/con
 python -m pip install git+https://github.com/litex-hub/conda-build-prepare@v0.1.1#egg=conda-build-prepare
 
 # ANACONDA_USER isn't available in cross-repository PRs
-if [ "$ANADONDA_USER" != "" ]; then
+if [ "$ANACONDA_USER" != "" ]; then
```

Comment:
Can you not get it from `$GITHUB_REPOSITORY`?

Reply:
No because now the user/organization in `$GITHUB_REPOSITORY` is `hdl` and there's even no such channel on Anaconda.

Reply:
That's specifically to handle the "repo on hdl, packages on litex-hub" case